logistics_planning_sale/models/sale_order_line.py

```diff
@@ -39,6 +39,14 @@ class SaleOrderLine(models.Model):
         are allowed for this line
         """,
     )
+    logistics_vehicle_type_id = fields.Many2one(
+        'vehicle.type',
+    )
+
+    @api.onchange('logistics_vehicle_type_id')
+    def _onchange_logistics_vehicle_type_id(self):
+        if self.logistics_vehicle_type_id:
+            self.ls_transport_type = self.logistics_vehicle_type_id.ls_sale_transport_type
```

Comment:
This code (adapted to existing field), should be moved to `logistics_planning_mgmt_weight` due to needed dependencies.

Reply:
This code is been lost, isn't it?

Reply:
This code already existed
https://github.com/solvosci/slv-stock/blob/9d9452d9a94f0f030d2b89a4c77c287d6ad11601/logistics_planning_mgmt_weight/models/sale_order_line.py#L10-L13